src/NinjaTools.FluentMockServer.API/Models/RequestMatcher.cs

```diff
@@ -2,14 +2,47 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Text;
 using JetBrains.Annotations;
 using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
 
 namespace NinjaTools.FluentMockServer.API.Models
 {
-  [DebuggerDisplay("{" + nameof(Path) + "} ({" + nameof(Method) + "})" )]
-  public class RequestMatcher
+    [DebuggerDisplay("{DebuggerDisplay()}")]
+    public class RequestMatcher
     {
+        public string DebuggerDisplay()
+        {
+            var path = Path + QueryString ?? "";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'path'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)